drivers/adxl345/include/adxl345_params.h

```diff
@@ -53,9 +53,9 @@ extern "C" {
 #ifndef ADXL345_PARAM_OFFSET
 #define ADXL345_PARAM_OFFSET        { 0, 0, 0 }
 #endif
-#ifndef ADXL345_PARAM_SCALE_FACTOR
-#define ADXL345_PARAM_SCALE_FACTOR  (3.9)
-#endif
+
+#define ADXL345_PARAM_SCALE_FACTOR  (4)
```

Comment:
If this is not parametrizable anymore, this define should move somewhere else, in adxl345.c

Reply:
@cgundogan can you complete the code review (incl. related label steps)?